docs/user/rstudio/ide/guide/code/diagnostics.qmd

```diff
@@ -33,51 +30,51 @@ Similarly, missing arguments (alongside extra, or missing, commas) are reported:
 
 ![](images/error-missing-comma.png){fig-alt="A screenshot of R code missing commas after each line, which returns an error 'exepcted , after expression'"}
 
-**Warn if variable used has no definition in scope**
+### Warn if variable used has no definition in scope
 
 Warn if a symbol is used with no definition in the current, or parent, scope. The diagnostics engine will supply a suggestion if there appears to be a typo in the symbol's name; that is, if a symbol with a similar name exists in scope as well.
 
 ![](images/error-missing-symbol.png){fig-alt="A screenshot of R code with a defined object not found, with a suggestion to use the correct object."}
 
-**Warn if variable is defined but not used**
+### Warn if variable is defined but not used
 
 This diagnostic helps to identify is a variable is created but never used. This can be helpful when attempting to clean up old code, or in diagnosing other errors (wherein you believe a particular variable should be used, but isn't).
```

Comment:
This diagnostic helps to identify **_if_** a variable

Reply:
fixed.